backend/routes/PostManagement/PostDeletion.py

```
@@ -0,0 +1,28 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+from api.v1.endpoints.Auth.auth import get_current_user
+from models.user import User
+from database.session import SessionLocal
+from sqlalchemy.orm import Session
+from services.services import get_post_by_id
+
+router = APIRouter()
+
+def get_db():
```

Ruff (0.8.2)

Missing return type annotation for public function `get_db`

(ANN201)